src/resources/shared/openshift/setup_azure.md

```diff
@@ -0,0 +1,9 @@
+### Setup Your Azure Profile
+
+Create a service principal and configure its access to Azure resources. Output the result in compatible with Azure SDK auth file and
+store it on `my.auth` file. Please refer to official
+[documentation](https://docs.openshift.com/container-platform/4.10/installing/installing_azure/preparing-to-install-on-azure.html) for details.
```

Comment:
```suggestion
OpenShift on Azure documentation](https://docs.openshift.com/container-platform/4.10/installing/installing_azure/preparing-to-install-on-azure.html) for details.
```

Link texts should be descriptive.

Reply:
I think its implicit since the write up is about OCP on Azure. Anyways, I have changed it.